packages/cli/src/util/telemetry/commands/integration/index.ts

```diff
@@ -0,0 +1,31 @@
+import { TelemetryClient } from '../..';
+
+export class IntegrationTelemetryClient extends TelemetryClient {
+  trackCliSubcommandAdd(actual: string) {
+    this.trackCliSubcommand({
+      subcommand: 'add',
+      value: actual,
+    });
+  }
+
+  trackCliSubcommandList(actual: string) {
+    this.trackCliSubcommand({
+      subcommand: 'list',
+      value: actual,
+    });
+  }
+
+  trackCliSubcommandMove(actual: string) {
+    this.trackCliSubcommand({
+      subcommand: 'move',
+      value: actual,
+    });
+  }
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏remove?

Will we need this in the near future?